ui/ui-config/pages/index.tsx

```diff
@@ -1,47 +1,104 @@
+import { useRouter } from "next/router";
+import { useApi } from "../../ui-components/hooks/useApi";
 import Head from "next/head";
-import { Row, Col } from "react-bootstrap";
-import Link from "next/link";
-import { useApi } from "@grouparoo/ui-components/hooks/useApi";
-import { Actions } from "@grouparoo/ui-components/utils/apiData";
+import { Row, Col, Image, Button } from "react-bootstrap";
+import { useState, useEffect } from "react";
+import Loader from "../../ui-components/components/loader";
+
+export default function Page(props) {
+  const router = useRouter();
+  const { navigationMode, navigation } = props;
+  const [shouldRender, setShouldRender] = useState(false);
+  const [CTAs, setCTAs] = useState({
+    CTALink: "/session/sign-in",
+    CTAMessage: "Sign In",
+    CTATarget: null,
+  });
+  let currentStep = null;
+
+  useEffect(() => {
+    if (navigationMode === "config:authenticated") {
+      getSetupSteps(props);
+    } else {
+      setShouldRender(true);
+    }
+
+    async function getSetupSteps(props) {
+      console.log(`fetching steps`);
```

Comment:
![](https://media.giphy.com/media/PccxLzGfUAw48/giphy.gif)

Reply:
😅 IN MY DEFENSE.... I was pushing this to get feedback on testing, not to merge.  But yeah.